daemon/src/state.rs

```diff
@@ -938,6 +939,24 @@ where
     }
 }
 
+/// Returns the list of [`SomeIdentity`]s for the local peer.
+///
+/// # Errors
+///
+///   * Retrieving the project entities from the store fails.
+pub async fn list_identities<S>(peer: &Peer<S>) -> Result<Vec<SomeIdentity>, Error>
+where
+    S: Clone + Signer,
+{
+    peer.using_storage(move |store| {
+        let projects = identities::any::list(store)?
+            .filter_map(Result::ok)
```

Comment:
I'm wary of the error cases being hidden here. Something like this will lead to questions like, "How come this Urn was never announced?", due to it being malformed or an underlying bug. I would say we either outright `Error` here or we add some logging. I think I'm leaning more towards the former.

Reply:
It's a clone of `list_projects` and `list_users`, they too skip these errors. Should I fix them too or is it a special case?

Reply:
Ah fair, it's been a while since I looked at this code :innocent: I would say it's fine then as this is likely to be revisited regardless due to the upcoming RFCs =]